src/ui/colorThemes.ts

```
@@ -0,0 +1,74 @@
+// Helper function to get border color for each theme
+export function getBorderColor(theme: ColorTheme): string {
+  switch (theme) {
+    case 'blue':
+      return 'blue'; // Blue theme
+    case 'green':
+      return 'green'; // Green theme
+    case 'orange':
+      return 'yellow'; // Orange theme (closest terminal color)
+    case 'purple':
+      return 'magenta'; // Purple theme
+    case 'classic':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)